tools/clang/test/DXC/Inputs/bom-main-ascii.hlsl

```diff
@@ -0,0 +1,20 @@
+// Tests main and include files with/without BOM to ensure BOM is stripped.
```

Comment:
Are these files really just additions?  Aren't they coming from somewhere else?  If so, shouldn't they be moved instead to preserve the ability to trace them back in history?

Reply:
How to move but not remove the original file?
I hope to keep the original file and remove them when hctcmds test disabled.

Reply:
Could you update hctcmds to refer to the file in a new location?